### PR TITLE
Add spam assassin footer note

### PR DESCRIPTION
**Pull request by joshenlim**
To help clarify the warnings of the recently added spam validation - this validation is powered by Spam Assassin and not something built in house, so will need users to refer to spam assassin for any warnings
image

### CODE REVIEW

File: apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx [hunk 1]
```diff
@@ -29,28 +30,30 @@ export const SpamValidation = ({ validationResult }: SpamValidationProps) => {
...
-                  </Table.tr>
-                ))}
-              />
+              <div className="flex flex-col gap-y-3">
+                <p>
+                  {hasSpamWarning
+                    ? ` Rectify the following issues to improve your email's deliverability in order of priority:`
+                    : ` Address the following issues to improve your email's deliverability:`}
+                </p>
+                <Table
```
Comment (MildTomato) [inline apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx:40]: Can we use Table_shadcn_ ?

**Review by MildTomato — COMMENTED** [review]
see comment

File: apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx (hunk 1 above)
Comment (MildTomato) [reply apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx:40]: https://design-system-supabase.vercel.app/design-system/docs/components/table

Comment (joshenlim) [reply apps/studio/components/interfaces/Auth/EmailTemplates/SpamValidation.tsx]: done!

**Review by saltcod — APPROVED** [review]

**Pull request merged**